### DISCUSSION TITLE
Doesn't rspress support react@17?

### DISCUSSION THREAD

**Discussion by AnsonKong** [web-infra-dev/rspress#895] [Q&A] [answered]
rspress 不能支持 react@17 吗？

**Comment by AnsonKong** [comment]
我在迁移 dumi 到 rspress，按照指南的手动安装之后，yarn dev 页面空白报错，然后我升级 react 到 18 后则正常。
![image](https://github.com/web-infra-dev/rspress/assets/16773512/25a3cb76-713b-40fd-bcfc-749b9013c5f3)

**Comment by Timeless0911** [comment, marked as answer]
是的，不支持 react 17
